lms/static/sass/multicourse/_courses.scss

```diff
@@ -257,160 +333,152 @@ $facet-background-color: #007db8;
-      font-family: $sans-serif;
+  .search-facets-lists section {
+    border-top: 1px solid $courseware-button-border-color;
+  }
+
+  .facet-list {
+    @extend %ui-no-list;
+    @include clearfix();
+    padding-bottom: ($baseline/2); 
+  }
+
+  .facet-option {
+    @include float(left);
+    @include line-height(18.92);
+    @include transition(all $tmg-f2 ease-out 0s);
+    @include clearfix();
+    @extend %t-action3;
+    @extend %text-truncated;
+    border-radius: 0px;
+    opacity: 1;
+    width: 100%;
+    color: $facet-text-color;
+
+    &.collapse {
+      max-height: ($baseline*14);
     }
 
-    section {
-      margin: 0;
-      padding: ($baseline/2) 0;
+    li {
+      @include clearfix();
+      position: relative;
+      clear: both;
+      padding: 0;
+      height: ($baseline*1.5);
     }
 
-    .facet-option {
-      @include float(left);
+    .count {
+      @include right($baseline*0.6);
+      @include float(right);
       @include box-sizing(border-box);
+      @include transition(all 0.2s ease-out);
       @include line-height(18.92);
-      @include transition(all $tmg-f2 ease-out 0s);
-      @extend %t-action3;
-      opacity: 1;
-      width: 100%;
-      overflow: hidden;
-      text-overflow: ellipsis;
-      white-space: nowrap;
-      color: $facet-text-color;
-
-      //STATE: hover, visited
-      &:hover,
-      &:visited {
-        color: $facet-text-color;
-        text-decoration: none;
-      }
     }
 
-    ul {
-      margin: 0;
-      padding: 0;
-      overflow: hidden;
-      list-style: outside none none;
+    //STATE: hover
+    &:hover {
+      background: $facet-background-color;
+      color: $white;
+      text-decoration: none;
 
-      &.collapse {
-        max-height: ($baseline*14);
+      .count {
+        color: $white;
       }
+    }
+  }
 
-      li {
-        @include clearfix();
-        @include line-height(18.92);
-        @extend %t-icon6;
-        position: relative;
-        clear: both;
-        border-top: 1px solid $white;
-        padding: 0;
-        height: ($baseline*1.5);
-        overflow: hidden;
-
-        .count {
-          @include right($baseline*0.6);
-          @include box-sizing(border-box);
-          @include transition(all 0.2s ease-out);
-          @include line-height(18.92);
-          position: absolute;
-        }
-
-        //STATE: hover
-        &:hover {
-          background: $facet-background-color;
-          color: $white;
-          text-decoration: none;
+  .toggle {
+    @include clearfix();
 
-          .count,
-          .facet-option {
-            color: $white;
-          }
-        }
-      }
+    button {
+      @extend %t-icon6;
+      @extend %t-strong;
+      color: $facet-background-color;
     }
+  }
+}
 
-    .search-facets-lists section {
-      border-top: 1px solid $courseware-button-border-color;
-    }
+// +All Other Styles
+// ====================
+.find-courses, .university-profile {
+  background: $course-profile-bg;
```

Comment:
This seems like a funny variable to use here ... can you use a plain color one or will a change here make sense throughout the system?